packages/transactions/src/__tests__/lifetime-test.ts

```diff
@@ -0,0 +1,151 @@
+import { Address } from '@solana/addresses';
+import { SOLANA_ERROR__TRANSACTION__NONCE_ACCOUNT_CANNOT_BE_IN_LOOKUP_TABLE, SolanaError } from '@solana/errors';
+import { Blockhash } from '@solana/rpc-types';
+import {
+    CompiledTransactionMessage,
+    CompiledTransactionMessageWithLifetime,
+    Nonce,
+} from '@solana/transaction-messages';
+
+import { getTransactionLifetimeConstraintFromCompiledTransactionMessage } from '../lifetime';
+
+const SYSTEM_PROGRAM_ADDRESS = '11111111111111111111111111111111' as Address;
+const U64_MAX = 2n ** 64n - 1n;
```

Comment:
nit: Do you want to make the same change here?

Reply:
I don't have a strong opinion either way, but I think this reads a bit clearer about the intention for the tests which was why I didn't change it here.